itests/catalog.feature

```diff
@@ -1,30 +1,29 @@
 Feature: catalog testing
 
 Scenario: java catalog list
-  When command('jbang catalog add tc jbang-catalog.json')
+  When command('jbang catalog add --global tc jbang-catalog.json')
```

Comment:
why is --global now needed here ?

Reply:
ah to avoid any conflicts with jbang catalogs in jbang itself.

Reply:
Yeah :-)

It would probably be nice if tests were run from a clean CWD but that would probably mess up the tests that need extra files.